### DISCUSSION TITLE
Performance of the eww

### DISCUSSION THREAD

**Discussion by timsofteng** [elkowar/eww#616] [General]
Hello. I use sway with waybar. Almost all my soft is as lightweight as possible except of browser. Will I get the same performance with eww as with waybar throw same widgets or eww is more resource hungry?

**Comment by viandoxdev** [comment]
Eww is based on GTK, just like waybar, and written in rust so performance shouldn't be a problem. It should be pretty light unless you got a very complex config, say 100+ widgets and 30+ polling/listening variables which point you will notice lag and potentially some bugs (might have been fixed in recent commits).
